game/weather.rpy

```diff
@@ -0,0 +1,814 @@
+# Weather data
+default persistent.weather_api_key = None
+default persistent.weather_validate_apikey_in_time = None
+default persistent.is_weather_tracking_set_up = False
+default persistent.current_weather_type = weather.TYPE_CLEAR
+default persistent.current_weather_long = dict()
+
+# Location data
+default persistent.latitude = None
+default persistent.longitude = None
+default persistent.jn_hemisphere_north_south = None
+default persistent.jn_hemisphere_east_west = None
```

Comment:
I think just to make things consistent across the board and easy to identify/work with going forward, we'd be best served by prefixing all weather-related persistent variables with `jn_weather ` - especially since we have a lot of them here!

Apologies - forgot to add this earlier!